backend/src/main/java/gov/cdc/usds/simplereport/db/model/TestEvent.java

```diff
@@ -46,7 +46,7 @@ public class TestEvent extends BaseTestInfo {
   private TestOrder order;
 
   @JsonIgnore
-  @OneToMany(mappedBy = "testEvent", cascade = CascadeType.MERGE, fetch = FetchType.LAZY)
+  @OneToMany(mappedBy = "testEvent", cascade = CascadeType.MERGE)
```

Comment:
Is the default for fetch EAGER?
In [this doc](https://docs.oracle.com/javaee/6/api/javax/persistence/OneToMany.html) it looks like the default is to lazy load, but I'm not sure if I'm looking in the right place.

Reply:
Good catch, you're right - the default is lazy, so I changed to an explicit eager. Thanks!